src/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -110,6 +110,13 @@
     @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
     public int SAMPLE_SIZE=10000;
 
+    @Option(shortName = "INTERVALS", doc = "An interval list file that contains the positions to restrict the assessment. Please note that " +
+            "all bases of reads that overlap these intervals will be considered, even if some of those bases extend beyond the boundaries of " +
+            "the interval. The ideal use case for this argument is to use it to restrict the calculation to a subset of (whole) contigs. To " +
+            "restrict the calculation just to individual positions without overlap, please see CollectWgsMetricsFromSampledSites.",
+            optional = true, overridable = true)
```

Comment:
got it. thanks!

On Mon, Mar 7, 2016 at 12:52 PM, meganshand <notifications@github.com>
wrote:

> In src/java/picard/analysis/CollectWgsMetrics.java
> <https://github.com/broadinstitute/picard/pull/470#discussion_r55242198>:
>
> > @@ -110,6 +110,13 @@
> >      @Option(doc="Sample Size used for Theoretical Het Sensitivity sampling. Default is 10000.", optional = true)
> >      public int SAMPLE_SIZE=10000;
> >
> > +    @Option(shortName = "INTERVALS", doc = "An interval list file that contains the positions to restrict the assessment. Please note that " +
> > +            "all bases of reads that overlap these intervals will be considered, even if some of those bases extend beyond the boundaries of " +
> > +            "the interval. The ideal use case for this argument is to use it to restrict the calculation to a subset of (whole) contigs. To " +
> > +            "restrict the calculation just to individual positions without overlap, please see CollectWgsMetricsFromSampledSites.",
> > +            optional = true, overridable = true)
>
> So that it can be overridden by the INTERVALS argument to
> CollectWgsMetricsFromSampledSites. Without it if both tools have an
> INTERVALS argument and you get a command line parser error when you run
> CollectWgsMetricsFromSampledSites.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/broadinstitute/picard/pull/470/files#r55242198>.
>